Codeinwp/otter-blocks
# Icon List enhancement

Closes #824.


### Summary


New controls and options.

### Screenshots 

----

### Test instructions


1. Make an Icon List block with the current version of Otter
2. Install the version from this PR and check if the block is breaking.
3. Test new options.



---- 

### Checklist before the final review
